copyfiles.py

```diff
@@ -0,0 +1,90 @@
+import os
```

Comment:
In my previous commit I deleted it because this should be placed inside the util folder, I had a different file of my own there, that's why I didn't simply move it. Please move it to `utils/copyfiles.py`